samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/CallingBot.cs

```diff
@@ -142,6 +151,25 @@ private async Task NotificationProcessor_OnNotificationReceivedAsync(Notificatio
 
                 var recordingLocation = await teamsRecordingService.DownloadRecording(recording.RecordingLocation, recording.RecordingAccessToken);
 
+                try
+                {
+                    var callDetails = callService.Get(callId);
+                    var result = await speechService.ConvertWavToText(recordingLocation);
+
+                    if (result.Reason == ResultReason.RecognizedSpeech)
+                    {
+                        var threadId = (await callDetails)?.ChatInfo?.ThreadId;
+                        if (threadId != null)
+                        {
+                            await botService.SendToConversation($"You said: {result.Text}", threadId);
```

Comment:
Are bots that are in the call automatically in the chat as well?

Reply:
So there is a possibility that a user adds the app to a scheduled meeting, starts the call, then removes the app where this will fail. I've added an catch statement to explain what might happen. I can go further and include a call to Graph to see if it's installed, but in the Incident scenario we already ensure the app is installed in the scenario.